manly.py

```diff
@@ -155,4 +149,12 @@ def main(command):
     if not len(args.command):
         print("manly: missing COMMAND\n" "Try 'manly --help' for more information.")
```

Comment:
Consider adding a `sys.exit(0)` on the next line and remove the `else`-clause completely. I know this isn't really part of your changes, but "flat is better than nested" :)

Reply:
Done. :)